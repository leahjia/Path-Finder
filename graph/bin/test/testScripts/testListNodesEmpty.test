# DesignMap list of nodes testing file.
# Tests that a list of nodes in map can be computed with no nodes in the map

# Create a graph
CreateGraph map1

# Finds the nodes of map1
ListNodes map1
