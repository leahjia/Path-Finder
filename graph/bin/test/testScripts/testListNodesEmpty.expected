# DesignMap list of nodes testing file.
# Tests that a list of nodes in map can be computed with no nodes in the map

# Create a graph
created graph map1

# Finds the nodes of map1
map1 contains:
